test/benchmarks/qasm/54QBT_25CYC_QSE_3.qasm

```diff
@@ -1,7 +1,7 @@
 // Originally source from the QUEKO benchmark suite
 // https://github.com/UCLA-VAST/QUEKO-benchmark
 // A benchmark that is near-term feasible for Google Sycamore with a optimal
-// soluation depth of 25
+// solution depth of 25
```

Comment:
@1ucian0 @mtreinish  is it fine to change the benchmarks folder? I remember in the past we had to be careful about that

Reply:
It's a comment in a qasm... I think it is fine. @mtreinish ?

Reply:
Yeah this can be changed without issue.

The problem you're remembering is if we were to change the source code in a benchmark's python code. The asv tool we use for nightly benchmarks implicitly versions each benchmark as a hash of the source code, so if we tweak the source code slightly that will invalidate previous results because it treats it as a new benchmark version. But just tweaking the qasm files used for benchmarking won't cause that problem.